nvflare/private/fed/client/client_json_config.py

```diff
@@ -102,8 +102,8 @@ def process_config_element(self, config_ctx: ConfigContext, node: Node):
             self.current_exe.executor = self.authorize_and_build_component(element, config_ctx, node)
             return
 
-    def authorize_and_build_component(self, config_dict, config_ctx, node):
-        t = super().authorize_and_build_component(config_dict, config_ctx, node)
+    def build_component(self, config_dict):
```

Comment:
should we use `authorize_and_build_component` or `build_component`?

Reply:
at the entry point, use authorize_and_build_component(). build_component() is used by the recursive calls.